flexget/plugins/output/decompress.py

```diff
@@ -227,12 +218,12 @@ def handle_entry(self, entry, config):
 
                     log.verbose('Extracted: %s' % path )
                     success = True
-                except (IOError, os.error) as e:
+                except (IOError, os.error) as error:
                     error_message = 'OS error while creating file: %s (%s)' % \
-                                    (destination, e)
-                except (zipfile.BadZipfile, rarfile.Error) as e:
+                                    (destination, error)
+                except (zipfile.BadZipfile, rarfile.Error) as error:
                     error_message = 'Failed to extract file: %s in %s (%s)' % \
-                                    (path, archive_path, e)
+                                    (path, archive_path, error)
```

Comment:
We allow up to 120 line length, I'd like to avoid the `\` line extension where possible.